src/tools/genpolicy/rules.rego

```diff
@@ -1319,6 +1319,35 @@ ExecProcessRequest {
     print("ExecProcessRequest 3: true")
 }
 
+UpdateInterfaceRequest {
+    print("UpdateInterfaceRequest 1: input =", input)
+
+    i_interface := input.interface
+    p_allowed_raw_flags := policy_data.request_defaults.UpdateInterfaceRequest.allow_raw_flags
+
+    print("UpdateInterfaceRequest 1: i_interface.raw_flags =", i_interface.raw_flags)
+    print("UpdateInterfaceRequest 1: p_allowed_raw_flags =", p_allowed_raw_flags)
+
+    # The agent only checks certain raw_flags; reject anything not processed.
```

Comment:
I would prefer a comment similar to:

"Typically, just FLAG_ABC is allowed."

No need to comment about what the Agent does.